src/main/scala/com/fulcrumgenomics/umi/DuplexConsensusCaller.scala

```diff
@@ -94,15 +95,25 @@ object DuplexConsensusCaller {
   * @param trim if true, quality trim reads in addition to masking. If false just mask.
   * @param errorRatePreUmi the estimated rate of errors in the DNA prior to attaching UMIs
   * @param errorRatePostUmi the estimated rate of errors in the DNA post attaching UMIs
-  */
+  * @param minReads the minimum number of input reads to a consensus read (see [[CallDuplexConsensusReads]]).
+  * */
 class DuplexConsensusCaller(override val readNamePrefix: String,
                             override val readGroupId: String    = "A",
                             val minInputBaseQuality: PhredScore = DuplexConsensusCaller.MinInputBaseQuality,
-                            val trim: Boolean = false,
+                            val trim: Boolean                   = false,
                             val errorRatePreUmi: PhredScore     = DuplexConsensusCaller.ErrorRatePreUmi,
-                            val errorRatePostUmi: PhredScore    = DuplexConsensusCaller.ErrorRatePostUmi
+                            val errorRatePostUmi: PhredScore    = DuplexConsensusCaller.ErrorRatePostUmi,
+                            val minReads: Seq[Int]              = Seq(1)
                            ) extends UmiConsensusCaller[DuplexConsensusRead] with LazyLogging {
 
+  private val minTotalReads: Int = this.minReads.head
+  private val minXYReads: Int    = this.minReads.take(2).last
+  private val minYXReads: Int    = this.minReads.last
```

Comment:
I know I wrote something similar to this somewhere else.  Now though, I think it would be clearer as:

```scala
private val Seq(minTotalReads, minXyReads, minYxReads) = this.minReads.padTo(3, this.minReads.last)
```

Also can `XY` and `YX` be `Xy` and `Yx` please?  Or maybe we should just call them X and Y instead of XY and YX?